### PR TITLE
[10.0] account_multicurrency_revaluation: Fix tests using company from demo data

### PR DESCRIPTION
**Pull request by grindtildeath**

### CODE REVIEW

**Comment by grindtildeath** [comment]
No idea why runbot is red here :confused:

**Comment by ghost** [comment]
due to 
```
2018-07-04 09:45:52,240 178 WARNING openerp_test odoo.models: account.payment.method.create() includes unknown fields: company_id
```

File: account_multicurrency_revaluation/demo/account_demo.xml [hunk 1]
```diff
@@ -77,10 +117,18 @@
...
         <field name="name">A Sales Journal</field>
         <field name="code">SJ</field>
         <field name="type">sale</field>
+        <field name="company_id" ref="res_company_reval" />
+    </record>
+    <record id="account_payment_method_manual_in" model="account.payment.method">
+        <field name="name">Manual</field>
+        <field name="code">manual</field>
+        <field name="payment_type">inbound</field>
+        <field name="company_id" ref="res_company_reval" />
```
Comment (ghost) [inline account_multicurrency_revaluation/demo/account_demo.xml]: remove `<field name="company_id" ref="res_company_reval" />` to make runbot happy )

Comment (grindtildeath) [reply account_multicurrency_revaluation/demo/account_demo.xml]: Thanks :)

**Comment by fclementic2c** [comment]
@grindtildeath Can we merge this one now ?

**Comment by grindtildeath** [comment]
@fclementic2c For me it's good.
@oleksandrpaziuk Can you approve ?

**Review by ghost — APPROVED** [review]
LGTM!
Same as https://github.com/OCA/account-closing/pull/79 only for 10.0

**Pull request merged**